Android/app/src/main/res/values/strings.xml

```diff
@@ -105,8 +106,11 @@
     <string name="share_intent_text">Share Contact Details</string>
     <string name="share_menu">Share</string>
     <string name="share_contact_text">Share Contact</string>
+    <string name="cant_generate">Can\'t be generated. Need to download QR services</string>
+    <string name="add_contact">Added Contact</string>
+    <string name="cancel_contact">Cancelled Added Contact</string>
```

Comment:
the above three strings are not used anywhere.

Reply:
@Swati4star you need to be more specific. I check for unused strings in android studio and get no results. How do you know thay are unsused? Other people add code and i get conflicts that i need to resolve and merge my code. Then people add new toasts which i again resolve and merge.